app/views/alchemy/admin/attachments/_filter_bar.html.erb

```diff
@@ -0,0 +1,32 @@
+<% params_to_keep = [:tagged_with, :q, :content_id, :options] %>
+
+<div id="filter_bar">
+  <h2><%= Alchemy.t('Filter') %></h2>
```

Comment:
This is actually the header for the whole filter bar, hence the `h2` tag and the name? ;)

What you want, would be more like a `label` for the select box. I don't see any value in this, especially if there is only one select. 

Let's introduce this, if we have more selects, ok?